shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2539,6 +2539,60 @@ void WebContents::SetWebRTCIPHandlingPolicy(
   web_contents()->SyncRendererPrefs();
 }
 
+v8::Local<v8::Value> WebContents::GetWebRTCUDPPortRange(
+    v8::Isolate* isolate) const {
+  uint16_t min = 0, max = 0;
+  auto* prefs = web_contents()->GetMutableRendererPrefs();
+
+  min = prefs->webrtc_udp_min_port;
+  max = prefs->webrtc_udp_max_port;
+
+  gin_helper::Dictionary dict = gin::Dictionary::CreateEmpty(isolate);
+  dict.Set("min", static_cast<uint32_t>(min));
+  dict.Set("max", static_cast<uint32_t>(max));
+  return dict.GetHandle();
+}
+
+void WebContents::SetWebRTCUDPPortRange(gin::Arguments* args) {
+  uint32_t min = 0, max = 0;
+  gin_helper::Dictionary range;
+
+  if (!args->GetNext(&range) || !range.Get("min", &min) ||
+      !range.Get("max", &max)) {
+    gin_helper::ErrorThrower(args->isolate())
+        .ThrowError(
+            "Lack of arguments. Note that both min and max port number should "
+            "be provided and organized as an object");
+    return;
+  }
+
+  if ((0 == min && 0 != max) || max > UINT16_MAX) {
+    gin_helper::ErrorThrower(args->isolate())
+        .ThrowError(
+            "Invalid arguments. Note that either port number should be in (0, "
+            "65535] or both are 0");
+    return;
+  }
+  if (min > max) {
+    gin_helper::ErrorThrower(args->isolate())
+        .ThrowError(
+            "Invalid arguments. Note that min should not be greater than max");
+    return;
+  }
```

Comment:
Let's throw slightly terser errors here, to match our typical errors a bit more.

```suggestion
  if (!args->GetNext(&range) || !range.Get("min", &min) ||
      !range.Get("max", &max)) {
    gin_helper::ErrorThrower(args->isolate())
        .ThrowError("'min' and 'max' are both required");
    return;
  }

  if ((0 == min && 0 != max) || max > UINT16_MAX) {
    gin_helper::ErrorThrower(args->isolate())
        .ThrowRangeError("'min' and 'max' must be in the (0, 65535] range or [0, 0]");
    return;
  }
  if (min > max) {
    gin_helper::ErrorThrower(args->isolate())
        .ThrowRangeError("'max' must be greater than or equal to 'min'");
    return;
  }
```

Reply:
Sure~It seems better